.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "grpc/src/main/proto"]
+	path = grpc/src/main/proto
+	url = https://github.com/stargate/stargate-grpc-proto.git
+[submodule "testing/src/main/proto"]
```

Comment:
Is there a better way to share the same submodule in multiple directories?

Reply:
Why bother with an external repo and sub-modules? Why not include gRPC in the main `stargate` repo?

Reply:
If we want to keep gRPC code in a separate repo, it might be worth running a periodic CI job to check "latest against latest" compatibility. The gRPC repo will contain only its own CI, and local testing should be faster there.

Reply:
> Why bother with an external repo and sub-modules? Why not include gRPC in the main stargate repo?

Reply:
Could we publish `.proto` files as Maven artifacts?

Reply:
> Why bother with an external repo and sub-modules? Why not include gRPC in the main stargate repo?

Most of the code for gRPC is contained in the `stargate` repo. The external repo contains only proto (IDL) files describing the types and calls provided by the Stargate gRPC API. This allows:
* Client implementations to interface only with that proto repo to build new clients
* It allows the API to be versioned separately using tags

Reply:
Good point about versions :+1: 

At the same time we go `master` against `master` now (if I understand correctly)... Should we be using tags for the IDL from the beginning?

Reply:
> Could we publish .proto files as Maven artifacts?

Reply:
> Should we be using tags for the IDL from the beginning?

I'm iterating heavily on the API. I'll switch to tags when things stabilize.

Reply:
Created follow-up issue: https://github.com/stargate/stargate/issues/842

Reply:
sgtm. thx!